lib/cli/lib/helpers.js

```diff
@@ -17,6 +17,9 @@ export async function getVersion(npmOptions, packageName, constraint) {
   } else if (/storybook/.test(packageName)) {
     current = devDependencies[packageName];
   }
+  if (packageName === '@storybook/riot') {
```

Comment:
Why is this needed ?

Reply:
oops, should have deleted that. Was there to make the build pass